va/va_test.go

```diff
@@ -378,33 +379,58 @@ func TestHTTPTimeout(t *testing.T) {
 	// TODO(#1989): close hs
 
 	va, _ := setup(hs, 0)
-
 	setChallengeToken(&chall, pathWaitLong)
-	started := time.Now()
 
-	timeout := 50 * time.Millisecond
-	ctx, cancel := context.WithTimeout(context.Background(), timeout)
-	defer cancel()
+	testCases := []struct {
+		Name             string
+		SimplifiedVAHTTP bool
+	}{
+		{"Legacy VA HTTP", false},
+		{"Simplified VA HTTP", true},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.Name, func(t *testing.T) {
+			var expectMatch *regexp.Regexp
+			expectMatch = regexp.MustCompile(
+				"Fetching http://localhost:\\d+/.well-known/acme-challenge/wait-long: Timeout after connect")
+
+			if tc.SimplifiedVAHTTP {
+				err := features.Set(map[string]bool{"SimplifiedVAHTTP": true})
+				test.AssertNotError(t, err, "Failed to set SimplifiedVAHTTP feature flag")
+				defer features.Reset()
+				// Simplified VA HTTP error messages don't include the port # when it is
```

Comment:
I think there's a copy-paste error here (`#`)

Reply:
I meant "port #" as in "port number" but its probably clearer to just say "number".

Reply:
Fixed in c9dc957